modin/pandas/base.py

```diff
@@ -1106,11 +1098,26 @@ def _deprecate_downcast(self, downcast, method_name: str):
         return downcast
 
     def bfill(
-        self, *, axis=None, inplace=False, limit=None, downcast=lib.no_default
+        self,
+        *,
+        axis=None,
+        inplace=False,
+        limit=None,
+        limit_area=None,
+        downcast=lib.no_default,
     ):  # noqa: PR01, RT01, D200
         """
         Synonym for `DataFrame.fillna` with ``method='bfill'``.
         """
+        if limit_area is not None:
```

Comment:
Let's create issues for adding tests for new parameters of the methods updated.

Reply:
tests added.